snakemake_wrapper_utils/snakemake.py

```
@@ -11,14 +11,21 @@ def get_mem(snakemake, out_unit="MiB"):
+    # Return memory
+    if out_unit == "B":
+        return mem_mb * 1024 * 1024
+    elif out_unit == "KiB":
         return mem_mb * 1024
     elif out_unit == "MiB":
         return mem_mb
     elif out_unit == "GiB":
         return mem_mb / 1024
     else:
-        raise valueError("invalid output unit. Only KiB, MiB and GiB supported.")
+        raise valueError("invalid output unit. Only B, KiB, MiB and GiB supported.")
```

Ruff (0.11.9)

Undefined name `valueError`

(F821)